.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,3 +1,3 @@
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.0/apache-maven-3.6.0-bin.zip
```

Comment:
Possibly this PR branch needs to be rebased with master b/c some changes are already present there.

see https://github.com/takari/maven-wrapper/commit/b93c46869a626d95af1908713580d88fc349937c